ui/app/AppLayouts/Profile/views/ContactsView.qml

```diff
@@ -259,12 +259,12 @@ Item {
             onCountChanged: searchResults.isAddedContact = searchResults.isContactAdded()
 
             onContactClicked: {
-                root.store.changeAppSectionBySectionType(Constants.appSection.chat)
```

Comment:
Don't wanna bikeshed here, but wouldn't it still be preferred to have this API provided by the store to reduce dependencies in the view?

Reply:
`changeAppSectionBySectionType` is called by many places in the UI, hence making it a global function. If we move it to the store here, then each store should have it as well having us duplicating code. We could either do this, or keep it in the Global component which is it's single place to update if needed.